### ISSUE TITLE
TODOs

### ISSUE THREAD

**Issue by benhylau** [benhylau/mesh-workshop#1]
- [x] Figure out why mesh-orange conf.d processing fails when the tar.gz is not named 20-sample.tar.gz
- [x] Waiting on PRs to be merged into minimal-debian-builder and mesh-orange, then a new release on mesh-router-builder
- [x] Waiting on docker 18.02 release that [fixes ramdisk bug](https://github.com/benhylau/mesh-router-builder/pull/5#issuecomment-362804991)
